### PR TITLE
Upload buton aria-label fix

### PR DESCRIPTION
**Pull request by Mottie**
## Description

The `aria-label` on the upload file button gets its content from the `ui:title`. For our form 10182 (Notice of Disagreement) the title is a React component and thus the aria-label ends up including `[object Object]` as text (see screenshot). This PR switches the order of values, making the `schema.title` first.

Related: https://github.com/department-of-veterans-affairs/va.gov-team/issues/23526

## Testing done

Added unit test

## Screenshots

Screen Shot 2021-04-30 at 1 44 26 PM

## Acceptance criteria
- [x] `aria-label` prioritizes `schema.title` over `ui:title`

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by bkjohnson — COMMENTED** [review]
What would happen if the title for form 10182 was a string instead of a React component?

**Comment by Mottie** [comment]
The design - https://vsateams.invisionapp.com/console/share/8Y10I6K7DU9R/600538587 - has the title as a header, so in this case it's an `h3`

**Comment by bkjohnson** [comment]
Can you point me to the `uiSchema` that you're talking about? I see [this](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/appeals/10182/config/form.js#L65) and some other `ui:title` assignments for various pages, but I'm not seeing an `<h3>` for a title.

**Comment by Mottie** [comment]
The `ui:title` is set by this file: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/file.js#L6

I'm passing a React content to the `fileUiSchema` in this file: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/appeals/10182/utils/upload.js#L23

and the title itself is here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/appeals/10182/content/EvidenceUpload.jsx#L11-L15

File: src/platform/forms-system/src/js/fields/FileField.jsx [hunk 1]
```diff
@@ -399,8 +399,8 @@ class FileField extends React.Component {
                     }
                   }}
                   tabIndex="0"
-                  aria-label={`${buttonText} ${uiSchema['ui:title'] ||
-                    schema.title}`}
+                  aria-label={`${buttonText} ${schema.title ||
+                    uiSchema['ui:title']}`}
```
Comment (bkjohnson) [inline src/platform/forms-system/src/js/fields/FileField.jsx:403]: Could we use a value like `titleString` here following `buttonText`?

```js

const titleString = typeof uiSchema['ui:title'] === 'string' ? uiSchema['ui:title]' : schema.title
```

**Review by bkjohnson — COMMENTED** [review]
Thank you for the links. The reason I'm (possibly) being annoying about this is because I'm a bit nervous about reversing the order since it could have bad consequences for forms which haven't been configured with this in mind. Instead of reversing the order, could we do a typecheck on `uiSchema['ui:title']` and only use it if it's a string, otherwise use `schema.title`?

File: src/platform/forms-system/src/js/fields/FileField.jsx (hunk 1 above)
Comment (Mottie) [reply src/platform/forms-system/src/js/fields/FileField.jsx:403]: No worries! I think your suggestion is reasonable. I'll make the change first thing in the morning.

**Review by bkjohnson — APPROVED** [review]
:+1:

**Pull request merged**